app/models/asset.rb

```diff
@@ -80,6 +80,7 @@ class Asset
     after_transition to: :uploaded do |asset, _|
       asset.save!
       asset.remove_file!
+      FileUtils.rmdir(File.dirname(asset.file.path))
```

Comment:
Could we make `asset.remove_file!` do this I wonder? Sad we have to put this in two places.

Reply:
I think we'd have to give a custom CarrierWave storage class, which uses a different file type.  Like https://github.com/alphagov/whitehall/blob/f65a2f354a717032cf1009ff951335513ca0b222/lib/whitehall/asset_manager_storage.rb, but doing less work obviously.

Reply:
Do you think we should do that?  It'd be a bit more than two lines.

Reply:
Sorry forgot to reply! Yeah, I think it's probably not worth it.